packages/docusaurus-plugin-sitemap/src/pluginOptionSchema.ts

```diff
@@ -5,19 +5,20 @@
  * LICENSE file in the root directory of this source tree.
  */
 import * as Joi from 'joi';
+import {EnumChangefreq} from 'sitemap';
 import {PluginOptions} from './types';
 
 export const DEFAULT_OPTIONS: Required<PluginOptions> = {
   cacheTime: 600 * 1000, // 600 sec - cache purge period.
-  changefreq: 'weekly',
+  changefreq: EnumChangefreq.WEEKLY,
   priority: 0.5,
   trailingSlash: false,
 };
 
 export const PluginOptionSchema = Joi.object({
   cacheTime: Joi.number().positive().default(DEFAULT_OPTIONS.cacheTime),
   changefreq: Joi.string()
-    .valid('always', 'hourly', 'daily', 'weekly', 'monthly', 'yearly', 'never')
+    .valid(...Object.values(EnumChangefreq))
```

Comment:
not very fan of enums (non-std js), and not alone
https://fettblog.eu/tidy-typescript-avoid-enums/
https://www.techatbloomberg.com/blog/10-insights-adopting-typescript-at-scale/

I think we'd rather avoid using them when possible, even if the underlying lib does use them

Reply:
Yeah, I also not a big fan of Enums, but the [String variety](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) isn't that bad as for example, homogenous Enums.

Of course I can also change back the types. Would you like to revert `EnumChangefreq` usage completely or only the Enum values spread?

Reply:
let's keep it, just hope libs will stop exporting enums in the future :)